tapestry.js

```diff
@@ -1212,32 +1213,35 @@ function updateViewedValue(id, amountViewedTime, duration) {
         
         // Save to database if logged in
         if (tapestryWpUserId) {
-            
-            if (id) {
-                var progData = {
-                    "post_id": tapestryWpPostId,
-                    "node_id": id,
-                    "progress_value": amountViewed
-                };
-                jQuery.post(TAPESTRY_PROGRESS_URL, progData, function(result) {})
-                .fail(function(e) {
-                    console.error("Error with adding progress data");
-                    console.error(e);
-                });
-            }
+            saveProgressCounter++;
+            // Send requests every 5 seconds
+            if (saveProgressCounter == TIME_BETWEEN_SAVE_PROGRESS) {
```

Comment:
Change this to check to see if it's been 5 seconds. Something like
```
var minTimeToSave = progressLastSaved.setSeconds(progressLastSaved.getSeconds() + TIME_BETWEEN_SAVE_PROGRESS); // e.g. 5 seconds after the last time saved 
if ( minTimeToSave < new Date()) {
```

Reply:
Will be comparing the time difference between current time and last saved time.